src/languages/en.ts

```diff
@@ -1726,12 +1726,14 @@ const translations = {
     languagePage: {
         language: 'Language',
         languages: {
-            en: {
-                label: 'English',
-            },
-            es: {
-                label: 'Spanish',
-            },
+            en: 'English',
```

Comment:
Doesn't it make sense to show each language in it's own source? i.e: Español for Spanish, no matter what language you're currently in?